src/Options/Foundry/Models/DeploymentsListOptions.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AzureMcp.Options.Foundry.Models;
+
+public class DeploymentsListOptions : GlobalOptions
+{
+    public string Endpoint { get; set; }
```

Comment:
Given we return continuation token in the list operation, should we consume it?

Reply:
Got it. I thought we returned continuation token to the client and let client decide if it needs next page.

In our interface, client can only specify max pages.

Do we document how many records per page?

If not, maybe `max records` makes more sense?

Reply:
Per Jeff's feedback:
Build something and then let's demo it. We can modify the code at any time.